### ISSUE TITLE
Why is NVM a requirement?

### ISSUE THREAD

**Issue by sunesimonsen** [zendesk/node-publisher#39]
I'm using https://github.com/Schniz/fnm, so I don't have NVM installed. But looking at the code, it doesn't look like we actually use it for anything.

**Comment by vecerek** [comment]
You're right, I think we just wanted to make sure that the `.nvmrc` file is respected. Probably a better thing to do would be to run `node -v` and compare the version to the one in `.nvmrc`.

I've opened a PR to remove the dependency (#40) and added node version validation in https://github.com/zendesk/node-publisher/pull/40/commits/b12d09fc79eea2d11cfcab27bf9e83628a051a51.
